articles/libraries/lock/v10/configuration.md

```diff
@@ -844,7 +844,7 @@ var options = {
 
 ### oidcConformant {Boolean}
 
-Lock should be used in OIDC Conformant mode when embedding it directly in your application. When this mode is enabled, it will force Lock to use Auth0's current authentication pipeline and will prevent it from reaching legacy endpoints. This mode is **not** required when using Lock at Auth0's [hosted login page](/hosted-pages/login).
+Lock should be used in OIDC Conformant mode when embedding it directly in your application. When this mode is enabled, it will force Lock to use Auth0's current authentication pipeline and will prevent it from reaching legacy endpoints. This mode is **not** required when implementing universal login with a [login page](/hosted-pages/login).
 

```

Comment:
Maybe in places like this we can say "implementing universal [login](/hosted-pages/login)" instead of 'with a login page'.